README.md

```diff
@@ -14,9 +14,345 @@ ManageIQ plugin for the Hawkular provider.
 
 ## Development
 
-See the section on pluggable providers in the [ManageIQ Developer Setup](http://manageiq.org/docs/guides/developer_setup)
+This is a quick guide outlining all the necessary steps to get ManageIQ up and
+running for development.
 
-For quick local setup run `bin/setup`, which will clone the core ManageIQ repository under the *spec* directory and setup necessary config files. If you have already cloned it, you can run `bin/update` to bring the core ManageIQ code up to date.
+### Dependencies
+
+* Ruby 2.3+ (Ruby 2.4.1 recommended)
+* Bundler 1.15.3+
+* PostgreSQL 9.5+
+* Memcached
+* Node/npm and the usual suspects: Bower, Yarn, Gulp, Webpack
+* Docker
+* 16gb RAM (8gb or less is doable, but not recommended)
+
+Any Linux/OS X System is compatible, as far as we are concerned.
+
+### Installing Dependencies
+
+#### Fedora
+
+First, install all system dependencies:
+
+```bash
+sudo dnf -y git-all memcached postgresql-devel postgresql-server postgresql-contrib bzip2 libffi-devel readline-devel sqlite-devel nodejs gcc-c++ libcurl-devel npm openssl-devel cmake openscap
```

Comment:
docker missing here for install?

Reply:
Isn't this all (or part of) already here? https://github.com/ManageIQ/guides/blob/master/developer_setup.md

Reply:
yes, I commented about it [here](https://github.com/ManageIQ/manageiq-providers-hawkular/pull/38#issuecomment-327071406)

Reply:
@josejulio but docker is missing in that dev guide because it is relevant for hawkinit but not for miq at the moment

Reply:
maybe it's still worth to have a guide of our own because we can update it as frequent as we want and make it tailored for our needs

Reply:
Also added a guide to Docker.